history-network.md

```diff
@@ -166,7 +166,20 @@ each receipt/transaction and re-rlp-encode it, but only if it is a legacy transa
 
 AccumulatorProof = Vector[Bytes32, 15]
 
-BlockHeaderProof = Union[None, AccumulatorProof]
+BeaconBlockBodyProof = Vector[Bytes32, 8] # Proof that EL block_hash in ExecutionPayload is part of BeaconBlockBody
+BeaconBlockHeaderProof = Vector[Bytes32, 3] # Proof that BeaconBlockBody root is part of BeaconBlockHeader
```

Comment:
What is the reason driving 2 separate proofs here rather than just one `BeaconBlockProof` that roots the `blockhash` in the `BeaconBlock` container (since [BeaconBlock](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#beaconblock) as extended by [this] (https://github.com/ethereum/consensus-specs/blob/dev/specs/bellatrix/beacon-chain.md#beaconblockbody) contains everything you need in one proof (so would save us one 2 values in the `HistoricalRootsBlockProof` datastructure.

Reply:
Yes, good point. I hadn't realized that `BeaconBlock` results in the same `hash_tree_root` as `BeaconBlockHeader`. So it looks like what you suggest will work. I will adapt.

Reply:
I've updated the PR with your suggestion.